assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -26,7 +26,9 @@ force_latency=cstate.id:1|3
 governor=performance
 energy_perf_bias=performance
 min_perf_pct=100
-
+{{if .PerPodPowerManagement}}
+enabled=false
```

Comment:
nit: Don't need to change this now, but the resulting profile would look better if the "enabled=false" was the only line in the [cpu] section. Have you confirmed that tuned handles this properly when several configuration options are provided for a plugin, followed by "enabled=false"?

Reply:
`Have you confirmed that tuned handles this properly when several configuration options are provided for a plugin, followed by "enabled=false"?`

Yes, the plugin is disabled.

`nit: Don't need to change this now, but the resulting profile would look better if the "enabled=false" was the only line in the [cpu] section. `

Right, if test fail I can update it, if passed I can do a refactor in the future

Reply:
done